pkg/apis/core/helper/helpers.go

```diff
@@ -39,6 +39,21 @@ func IsHugePageResourceName(name core.ResourceName) bool {
 	return strings.HasPrefix(string(name), core.ResourceHugePagesPrefix)
 }
 
+// IsHugePageResourceValueDivisible returns true if the resource value of storage is
+// integer multiple of page size.
+func IsHugePageResourceValueDivisible(name core.ResourceName, quantity resource.Quantity) bool {
+	pageSize, err := HugePageSizeFromResourceName(name)
+	if err != nil {
+		return false
+	}
+
+	if pageSize.Cmp(resource.Quantity{}) <= 0 || pageSize.MilliValue()%int64(1000) != int64(0) {
```

Comment:
`pageSize.Sign() <= 0` is a simpler way to check for negative or zero values

Reply:
done.